integrations/standalone/tests/pageobjects/Annotations.ts

```diff
@@ -0,0 +1,48 @@
+import type { Locator, Page } from '@playwright/test';
+import { Button } from './Button';
+import { Collapsible } from './Collapsible';
+import { Table } from './Table';
+
+export class Annotations {
+  readonly collapsible: Collapsible;
+  readonly table: Table;
+  readonly add: Button;
+  readonly delete: Button;
+
+  constructor(page: Page, parentLocator: Locator) {
+    this.collapsible = new Collapsible(page, parentLocator, { label: 'Annotations' });
+    this.table = new Table(this.collapsible.locator);
+    this.add = new Button(this.collapsible.locator, { name: 'Add row' });
+    this.delete = new Button(this.collapsible.locator, { name: 'Delete annotation' });
+  }
+
+  async expectToHaveValues(...annotations: Array<string>) {
+    await this.collapsible.open();
+    await this.table.expectToHaveValues(...annotations.map(anno => [anno]));
+  }
+
+  async fillValues(annotations: Array<string>) {
+    await this.collapsible.open();
+
+    const rowCount = await this.table.rows.count();
+    if (rowCount < annotations.length) {
+      for (let i = 0; i < annotations.length - rowCount; i++) {
+        await this.add.locator.click();
+      }
+    } else if (rowCount > annotations.length) {
+      await this.table.row(annotations.length).locator.click();
+      for (let i = 0; i < rowCount - annotations.length; i++) {
+        await this.delete.locator.click();
+      }
+    }
+
+    for (let i = 0; i < annotations.length; i++) {
+      await this.table.row(i).locator.locator('input').fill(annotations[i]);
+    }
+
+    const lastRow = this.table.rows.last().locator('input');
+    if (await lastRow.isVisible()) {
+      await lastRow.blur();
+    }
```

Comment:
This is needed because the `updateData` function of `TableMeta` only gets called `onBlur`.
Because of this, a user that edits a cell and then leaves the editor without bluring the cell will loose their changes. Any reason why we should not change the `updateData` function to be called `onChange` instead of `onBlur` in the ui-components?

Reply:
I can't tell you 100% the reason why I made this decision. 
I know that I've encountered problems when I've used an edit cell that updates on change, but maybe this was only because I didn't know why this problem occurred or how I could fix it. 
But currently, all these edit tables work like this, they only update onBlur.

So if you want to test if you can change it, please either add your own InputCell here or add an option to the one in the ui-components to define the behavior.